doc/usage/options.rst

```diff
@@ -44,6 +44,10 @@ tab.
         external key files linked to password entries are missing. This option
         is useful when you use the same KeePass database on multiple computers.
 
+
+    -   **Disable SSH key decryption progress bar**: (default: disabled) Do not
+        show a progress bar during SSH key decryption process.
```

Comment:
I think it would be helpful to mention that the UI will still be frozen/unresponsive while the keys are being decrypted.

Reply:
@dlech , done